src/App.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component } from 'react';
+import * as firebase from 'firebase';
 import './App.css';
 
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      users: 10
+    };
+  }
+
+  componentDidMount() {
+    const rootRef = firebase.database().ref();
+    const userRef = rootRef.child('user');
+    userRef.on('value', snap => {
+      this.setState({
+        users: snap.val()
+      });
+    });
+  }
+
   render() {
```

Comment:
@scottcrawford03 Thanks for chiming in! Always great to get additional feedback. Deconstructing that attribute is a great idea. I'm going to go in that direction.